Dockerfile

```diff
@@ -77,6 +76,12 @@ COPY . "$BUILD_DIR"
 
 # Build guacamole-server from local source
 RUN yum -y install $BUILD_DEPENDENCIES         && \
+    curl -L https://codeload.github.com/LibVNC/libvncserver/tar.gz/LibVNCServer-0.9.11 | tar -zx && \
```

Comment:
I got that link by browsing github... but I think this URL might be more appropriate: https://github.com/LibVNC/libvncserver/archive/LibVNCServer-0.9.11.tar.gz